client/src/main/scala/org/http4s/client/package.scala

```diff
@@ -37,11 +38,24 @@ package object client {
   implicit class WithBodySyntax(val method: Method with PermitsBody) extends AnyVal with EntityRequestGenerator
   implicit class NoBodySyntax(val method: Method with NoBody) extends AnyVal with EmptyRequestGenerator
 
+  // implicit val taskFunctor = new Functor[Task] {
+  //   def map[A, B](fa: Task[A])(f: A => B): Task[B] = fa.map(f)
+  // }
+
+  implicit val taskMonad = new Monad[Task] {
```

Comment:
cats-fs2 provides this already, and we already depend on it.

Sorry if this is a repeat, on flaky wifi.